docs/src/methods/fourier_surrogates.md

```diff
@@ -43,10 +48,101 @@ s = surrogate(ts, PartialRandomization(0.5))
 surroplot(ts, s)
 ```
 
+In addition to [`PartialRandomization`](@ref), we provide two other algorithms for producing partially randomized surrogates, outlined below.
+
+### Relative partial randomization
+
+The [`PartialRandomization`](@ref) algorithm corresponds to assigning entirely new phases to the Fourier spectrum with some degree of randomness, regardless of any deterministic structure in the original phases. As such, even for $\alpha = 0$ the surrogate time series can differ drastically from the original time series.
+
+By contrast, the [`RelativePartialRandomization`](@ref) procedure draws phases as:
+
+$$\phi \to \phi + \alpha \xi, \quad \xi \sim \mathcal{U}(0, 2\pi).$$
+
+With this algorithm, phases are progressively corrupted by higher values of $\alpha$: surrogates are identical to the original time series for $\alpha = 0$, equivalent to random noise for $\alpha = 1$, and retain some of the structure of the original time series when $0 < \alpha < 1$. This procedure is particularly useful for controlling the degree of chaoticity and non-linearity in surrogates of chaotic systems.
+
+### Spectral partial randomization
+
+Both of the algorithms above randomize phases at all frequency components to the same degree.
+To assess the contribution of different frequency components to the structure of a time series, the [`SpectralPartialRandomization`](@ref) algorithm only randomizes phases above a frequency threshold.
+The threshold is chosen as the lowest frequency at which the power spectrum of the original time series drops below a fraction $1-\alpha$ of its maximum value (such that the power contained above the frequency threshold is a proportion $\alpha$ of the total power, excluding the zero frequency).
+
+Click through the figures below to compare the three algorithms for various values of $\alpha$.
+
```

Comment:
Hm, I fear this may not be sustainable @brendanjohnharris . What happens if these links you use dissapear somehow? Or if someone builds the docs locally? I understand and appreciate that it took you a lot of effort to make this construction, but perhaps a really simple figure comparing the two algorithms that can be generated by an `@example` codeblock fits the bill better.

Reply:
This is already enough I would wager:
![image](https://github.com/JuliaDynamics/TimeseriesSurrogates.jl/assets/19669089/9c3e0f3f-59b1-4a48-b6b9-19fb1ef52b41)

the code generating the figure would have `a` as the parameter of courtse, so a user could always make the figure again with different a.

I've just pushed a commit that will allow you to use `surroplot!(gl, x, method)` with `gl = GridLayout(fig[:, 1])` or `2`.

Reply:
so that you can have the two algorithms side by side

Reply:
Sorry, correct the above: you should have three columns for the three algorithms, not two columns...

Reply:
Actually, forget everything I said: there is no real reason to use `surroplot` at all!!!

@brendanjohnharris please do me a favor and do a 3 by 3 plot: each row is an algorithm, each column is a value of `a`. We only need to plot the surrogates, no need to plot hte spectra / autocor,.